README.md

```diff
@@ -31,9 +31,9 @@ To add a new brand:
 
 # Publishing
 
-Create a git tag starting with `v`, e.g. `v1.2.3` and push the tag to GitHub. 
-
-Creating this tag will trigger a Travis CI to build, sign, and properly version the build. The release-ready APKs are available for side-loading from [GitHub Releases](https://github.com/medic/medic-android/releases) and are uploaded to the Google Play Console in the "alpha" channel. To release to the public, click "Release to Production" or "Release to Beta" via the Google Play Console for each flavor.
+1. Create a git tag starting with `v`, e.g. `v1.2.3` and push the tag to GitHub. 
+2. Creating this tag will trigger a Travis CI to build, sign, and properly version the build. The release-ready APKs are available for side-loading from [GitHub Releases](https://github.com/medic/medic-android/releases) and are uploaded to the Google Play Console in the "alpha" channel. To release to the public, click "Release to Production" or "Release to Beta" via the Google Play Console for each flavor.
```

Comment:
> properly version the build

So that's why I found no significant versioning in code... 

> To release to the public, click "Release to Production" or "Release to Beta" via the Google Play Console for each flavor.

Do devs do this or it's done by project maintainers, on a case-by-case basis?

Reply:
I've split that step into two - one for the dev to publish to the alpha channel, and one for the project manager to push to beta/prod.